store/comments.js

```diff
@@ -111,10 +116,17 @@ export const actions = {
   create ({dispatch}, data) {
     return this.app.$api.service('comments').create(data)
   },
-  patch ({dispatch}, data) {
+  patch ({dispatch, commit}, data) {
     return this.app.$api.service('comments').patch(data._id, data)
+      .then(() => {
+        commit('set', data.content)
+      })
   },
-  remove ({dispatch}, id) {
+  remove ({commit}, id) {
+    commit('removeComment', id)
     return this.app.$api.service('comments').remove(id)
+      // .then(() => {
+      //   commit('removeComment', id)
+      // })
```

Comment:
No code comments

Reply:
thanks for the feedback, in fact the comment is added by visual studio so i did not added backslash to the three lines. but it is a good point as it gives more visibility to others.  regarding v-input, it was what was partially causing the problem. v-model is already doing a bi-directional binding (https://vuejs.org/v2/guide/forms.html) it is bind with newContent. so updating the value of newContent will update the input field.

Reply:
comments have been removed on last commit